{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "a6d76ead_b2824250",
        "filename": "dump/dump_manager.cpp",
        "patchSetId": 16
      },
      "lineNbr": 100,
      "author": {
        "id": 1000089
      },
      "writtenOn": "2023-01-04T12:17:11Z",
      "side": 1,
      "message": "you will not get this and other errors below as part of SdBusError. SdBusError need to be base class of below exceptions so that it can come here, which is not the case. Please test. You can use std::exception though.\n\n```\nstruct Disabled final :public sdbusplus::exception::generated_exception\nstruct generated_exception : public exception\n\nclass SdBusError final : public internal_exception\n```",
      "range": {
        "startLine": 100,
        "startChar": 0,
        "endLine": 100,
        "endChar": 44
      },
      "revId": "7200005ceb5811d675e5750764434609d8893394",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}